src/commands/hosting-site-create.ts

```diff
@@ -1,21 +1,76 @@
-import { bold } from "cli-color";
+import { bold, yellow } from "cli-color";
 
+import { logLabeledSuccess } from "../utils";
 import { Command } from "../command";
-import { createSite } from "../hosting/api";
+import { Site, createSite } from "../hosting/api";
+import { promptOnce } from "../prompt";
 import { FirebaseError } from "../error";
 import { requirePermissions } from "../requirePermissions";
 import * as getProjectId from "../getProjectId";
 import * as logger from "../logger";
 
-export default new Command("hosting:site:create <siteName>")
+const LOG_TAG = "hosting:site";
+
+export default new Command("hosting:site:create [siteName]")
   .description("create a Firebase Hosting site")
+  .option("--app <appId>", "specify an existing Firebase Web App AppID")
```

Comment:
```suggestion
  .option("--app <appId>", "specify an existing Firebase Web App ID")
```

App AppID just sounds so redundant. However, someone like @rachelsaunders might say that it's not, so I'd defer to her.

Reply:
done